scripts/build_scripts/xcodebuild.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Google LLC
```

Comment:
Added this file to this Android-only build PR because `build_testapps` refers to it and I thought it would be better to submit it now and alter it later than to prune `build_testapps.py` of all xcode references only to add them back in later.

It seems safer this way.